Quark_Sketch/src/arduino/arduino.h

```diff
@@ -33,10 +33,10 @@ void loop( void );
 
 typedef struct _ZephyrPinDescription
 {
-	uint8_t arduinoPin;
 	uint8_t zephyrPin1;
 	uint8_t zephyrPin2;
-	uint8_t analogPinNumber;
```

Comment:
Keeping this info from Dino here, "  zephyrPin2 was there for the Arduino 101/TinyTILE because some of the headers have two pins connected and thus each would have to be muxed individually.  ".